app.js

```diff
@@ -12,7 +14,11 @@ app.use(
     origin: `${process.env.REACT_APP_HOST}:${process.env.REACT_APP_PORT}`,
   }),
 );
+app.use(express.json());
+
+app.use('/clams', clamsRouter);
 
+// eslint-disable-line
```

Comment:
although it is probably annoying when eslint gets mad at the `console.log` line here let's keep the warning instead of adding an unnecessary line disable